Makefile.include

```diff
@@ -153,8 +153,12 @@ ifneq (0,$(shell test -d $(RIOTBOARD)/$(BOARD); echo $$?))
 endif
 
 # Use TOOLCHAIN environment variable to select the toolchain to use.
-# Default: gnu
+# Default for macOS: llvm; for other OS: gnu
+ifeq ($(BOARD)$(OS),nativeDarwin)
```

Comment:
please do this in two steps instead:
```make
ifeq ($(BOARD),native)
ifeq ($(OS),Darwin)
TOOLCHAIN ?= llvm
endif
endif
TOOLCHAIN ?= gnu
```
Since we set TOOLCHAIN with ?= we don't need to do it in an else statement, it will not overwrite the value set inside the if block.